### DISCUSSION TITLE
Need help porting web view app to 3.0

### DISCUSSION THREAD

**Discussion by JohnMajerle** [iTwin/itwinjs-core#2760] [Q&A]
Several weeks ago I created a viewer application using the delivered template:

        npx create-react-app myApp --template @itwin/web-viewer --scripts-version @bentley/react-scripts

I'd like update my app to the latest 3.0 npm packages.   Today I ran the above command in an empty directory but found that it hasn't been updated to 3.0 yet.  I was hoping to create a fresh app from scratch and simply graft onto it my from my existing app.

Is a 3.0 version of the source code for this template available?  

Thanks

John M.

**Comment by aruniverse** [comment]
Hey John, 3.0 hasn't been officially released so we don't have an official 3.0 iTwin Viewer template ready for the public to consume. We'll keep you posted in the coming weeks regarding the upgrade process and when we'll have a public template available for you to use.

**Reply by aruniverse** [reply]
See https://github.com/iTwin/itwinjs-core/discussions/2942
